src/components/Home/HomepageMap.tsx

```diff
@@ -74,17 +74,17 @@ export default function HomepageMap() {
                         position={station.location.coordinates.reverse()}
                         key={station.id}
                         stationId={station.id}
-                        iconImg={
+                        weatherDescription={
                             station.accuweather_location
                                 .current_weather_description
                         }
                         isDay={station.accuweather_location.isDayTime}
```

Comment:
Related to the comment in BaseMarker, but I think we don't need this isDay anymore. Any thoughts?